src/window.rs

```diff
@@ -132,6 +146,48 @@ impl Window {
         window
     }
 
+    /// Create the control panel
+    fn create_control_panel(
+        window: &winit::window::Window,
+        webview_id: Option<WebViewId>,
```

Comment:
Document should explain why this field needs option

Reply:
Do you mind explain this a bit? So I can document this better, I don't quite understand what this thing does

https://github.com/versotile-org/verso/blob/b17f53606c29dc1fd55b2edea1ef5b4ed44fb880/src/webview.rs#L45

Reply:
We have to init the pipeline ID first. I assume it can be anywhere.

Reply:
Should we put this in `Verso::new` instead?